src/helpers/index.js

```diff
@@ -1,30 +1,19 @@
 'use strict';
 
-import color from 'chartjs-color';
-
 import * as coreHelpers from './helpers.core';
 import * as canvas from './helpers.canvas';
+import * as color from './helpers.color';
 import * as curve from './helpers.curve';
 import * as dom from './helpers.dom';
 import * as easing from './helpers.easing';
 import * as options from './helpers.options';
 import * as math from './helpers.math';
 import * as rtl from './helpers.rtl';
 
-const colorHelper =
-	function(value) {
-		if (value instanceof CanvasGradient || value instanceof CanvasPattern) {
-			// TODO: figure out what this should be. Previously returned
-			// the default color
-			return value;
-		}
-
-		return color(value);
-	};
-
 export default {
 	...coreHelpers,
 	canvas,
+	color,
```

Comment:
I think destructuring the color helpers would be a good idea, so it stays as `helpers.color` and not `helpers.color.color`.

Reply:
Is that what we want for `getHoverColor` though? One thing we could do is just move `getHoverColor` to be a member of `DatasetController` since that's the only place it's used and then leave `color` as `helpers.color`. Thoughts?

Reply:
If we want to keep that backward compatible. I think it does not help at all to have the helpers categorized in the public interface. Internally its fine.